kratos/python_interface/application_importer.py

```diff
@@ -20,6 +20,7 @@ def ImportApplication(application, application_name, application_folder, caller)
         raise RuntimeError(msg)
     elif application_name not in Globals.RequestedApplications:  # This check is possibly redundant, as Python won't import the same module twice
         print("Importing    " + application_name)
+        sys.stdout.flush()
```

Comment:
I assume this is due to GiD not printing the output in the progress window? I would preffer not to put it in the `application_importer.py` since you are already adding one to your script.

At least not for the release, we can discuss later about this, but as I see this is something that GiD team has to solve, not us.

Reply:
I don't know if they "GiD" can solve that. This flush gives consistency to application imports and initialize. If not the message appears in other places and the user can not follow if the steps and the order of them is correct. What is the problem to keep it there?

Reply:
Is not a problem, just wanted to mention it so we can keep record ( I approved the PR!). 
I personaly feel that it is a bit tedious having to write a flush just because that window in particular is problematic. The script is perfectly fine, the problem is in the i/o buffer... But again is just a comment and probably we can open an issue to discuss that.

Reading the comment twice it sounds like I wanted it out, not the case.

Reply:
I did not want to use it there, but I could not find another solution. I just asked if it can be problem...because if it is, we remove it. I think that GiD will not solve that soon, it will take some time. I will ask tomorrow again. For them the problem is in our side.  So, maybe we can keep it there for the release,  I will not merge this change in the master.

Reply:
Ok to me 👍

Reply:
Regarding the print issue, would it be OK to remove the python print and add a similar one on `Kernel.AddApplication(application)`, using the kratos logging system? I understand that then it would be properly ordered with the remainder of the output (or the C part, at least).

I'd like to point that I do have objections to adding flushes to the code, since they force file I/O operations, which are a bottleneck in clusters. I do not think this one would be a big problem, though. @roigcarlo I agree that we might want to open an issue on this topic

Reply:
I have deleted it. A better solution was found.